homeassistant/components/sensor/nut.py

```diff
@@ -217,7 +217,9 @@ def opp_state(self):
             return STATE_TYPES['OFF']
         else:
             try:
-                return STATE_TYPES[self._data.status[KEY_STATUS]]
+                return " ".join(
+                    STATE_TYPES[state]
```

Comment:
Technically, all the possible types should be defined.  Although it also wasn't supposed to return multiple types together like it apparently is so there could certainly be a situation where we get something we weren't expecting.

I think only returning STATE_UNKNOWN if all types are unknown would be a good change.